src/main/resources/project/jython/update_app_config.jython

```diff
@@ -0,0 +1,65 @@
+#
+#  Copyright 2016 Electric Cloud, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+[% preamble %]
+
+print('Going for updating application')
+wl_username = r'''
+[% wl_username %]
+'''.strip()
+
+wl_password = r'''
+[% wl_password %]
+'''.strip()
+
+admin_url = r'''
+[% admin_url %]
+'''.strip()
+
+connect(wl_username, wl_password, admin_url)
+
+applicationName = r'''
+[% app_name %]
+'''.strip();
+
+planPath = r'''
+[% plan_path %]
+'''.strip()
+
+options = r'''
+[% options %]
+'''.strip()
+
+timeout = 300
```

Comment:
No need to default timeout here, it will be set to 300000 by weblogic itself if nothing is provided. Btw, these are ms.